extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -913,20 +913,22 @@ class AmpImageLightbox extends AMP.BaseElement {
 
       const rect = layoutRectFromDomRect(this.sourceImage_
           ./*OK*/getBoundingClientRect());
+      const imageBox = this.imageViewer_.getImageBox();
       const clone = this.sourceImage_.cloneNode(true);
+      clone.className = '';
       st.setStyles(clone, {
         position: 'absolute',
         top: st.px(rect.top),
         left: st.px(rect.left),
         width: st.px(rect.width),
         height: st.px(rect.height),
+        willChange: 'width, height, transform',
```

Comment:
We shouldn't animate `width` and `height`. Can we do `transform: scale()` instead?